VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-hearing-list/vho-hearing-list.component.html

```diff
@@ -4,10 +4,10 @@
 
     <div class="shareIcon"><app-copy-id [conference]="conference"></app-copy-id></div> 
     <div class="time listItem" id="{{ conference.id }}-time">{{ conference.scheduledDateTime | date: 'HH:mm' }}</div>
-    <div class="started listItemGreySmall" id="{{ conference.id }}-started">started</div>
-    <div class="startedTime listItemGreySmall" id="{{ conference.id }}-startedTime">{{ conference.startedDateTime | date: 'HH:mm' }}</div>
-    <div class="ended listItemGreySmall" id="{{ conference.id }}-ended">ended</div>
-    <div class="endedTime listItemGreySmall" id="{{ conference.id }}-endTime">{{ conference.endedDateTime | date: 'HH:mm' }}</div>
+    <div *ngIf="conference.startedDateTime" class="started listItemGreySmall" id="{{ conference.id }}-started">started</div>
```

Comment:
It would be better to have a single ngIf for startedDateTime and endedDateTime rather than twice.

Have you looked at ng-container

Something like

<ng-container *ngIf="conference.startedDateTime">
    <div>started text</div>
    <div>started time</div>
</ng-container>

Reply:
updated to ng-container